cv/qc-cv.obo

```diff
@@ -1,6 +1,6 @@
 format-version: 1.2
 data-version: 0.1.2
-date: 14:04:2021 12:39
+date: 21:07:2021 20:26
```

Comment:
can we (from now on) also increase the data-version, just to make sure people can pick the latest version without cross-checking with git/github?!

Reply:
Yes, good point!